### DISCUSSION TITLE
Testimonial - AMAZING tool

### DISCUSSION THREAD

**Discussion by sixindustries** [mjmlio/mjml#2883] [General]
Just want to give a huge shout out and kudos to those who have contributed to this app.  I can't believe how incredibly helpful and useful this thing is.  For years when marketing has asked me to code up a promotional email I was instantly filled with dread and procrastination due to the overwhelmingly frustrating task of HTML email.  But those days are OVER!

I learned the markup in literally a few minutes and was converting a design file into code with ease.  Bruh!!  Thank you!!!!!!!!!!!

Fantastic work you guys.

**Comment by iRyusa** [comment]
Thanks !
